build/prepare/CheckPrereqs.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="UbuntuCoreclrAndCoreFxDependencies.props" />
+
+	<Target Name="CheckPrereqs"
+			DependsOnTargets="CheckUbuntuAptGetDependencies;
+												CheckCentOsYumDependencies;">
+
+		<Message Text="Using Stage0 version:" />
+		<Exec Command="$(DotnetStage0) --version" />
+	</Target>
+
+	<Target Name="CheckUbuntuAptGetDependencies"
+			Condition=" '$(Rid)' == 'ubuntu.14.04-x64' ">
+
+		<Exec Command="dpkg -s %(UbuntuCoreclrAndCoreFxDependencies)" />
+
+		<Exec Command="dpkg -s %(DebianPackageBuildDependencies)" />
+
+		<OnError ExecuteTargets="ShowAptGetUsage" />
+	</Target>
+
+	<Target Name="ShowAptGetUsage">
+		<Message Text="Install the missing dependency with `sudo apt-get install {package}`." />
```

Comment:
````
Task "Exec"
  dpkg -s libc6
````

It shows that during the exec statement. If I get time I'll create a tooltask later. This is not a high value scenario for us and we do currently get sufficient diagnostics.